src/ClassicUO.Client/LegionScripting/API.py

```
@@ -24,293 +25,1385 @@ class Notoriety:
 def Virtue(virtue: str):
+    """
+     Use a virtue.
+     Example:
+     ```py
+     API.Virtue("honor")
+     ```
+    
+    """
     pass
 
 def NearestEntity(scanType: Any, maxDistance: int = 10):
```

Ruff (0.11.9)

Undefined name `Any`

(F821)